src/main/java/sqltoregex/property/regexgenerator/synonymgenerator/SynonymGenerator.java

```diff
@@ -135,7 +138,22 @@ public String generateRegExFor(S wordToFindSynonyms) {
 
     @Override
     public Set<A> getSettings() {
```

Comment:
will it be necessary to have more then 2 partners?
e.g.:
AVG ↔ Average ↔ Mittelwert ?

could be a possible Use Case in my eyes.